cluster-autoscaler/core/scaledown/actuation/actuator_test.go

```diff
@@ -349,6 +361,49 @@ func TestStartDeletion(t *testing.T) {
 				"drain-node-1": {ResultType: status.NodeDeleteOk},
 			},
 		},
+		"DS pods and deletion with drain": {
```

Comment:
Test case here and below are for testing code paths for when `IgnoreDaemonSetUtilization` is set to to `true` and `false`.